numaprom/udf/inference.py

```diff
@@ -81,6 +82,7 @@ def inference(_: str, datum: Datum) -> bytes:
     numalogic_conf = metric_config.numalogic_conf
 
     # Load inference model
+    set_aws_session()
```

Comment:
Also, instead of calling it in every udf, add it to load_model and save_model methods in tools.py